index.html

```diff
@@ -647,7 +647,7 @@ <h3>Proofs</h3>
           <dd>
 A string value that contains the data necessary to verify the digital proof
 using the `verificationMethod` specified. The contents of the value MUST be a
```

Comment:
Because it's relying on an informative reference, I think the strongest this can be is SHOULD, not MUST -- unless we replicate sufficient portion of the `[[?MULTIBASE]]` spec that this replication may be relied on for the MUST. Similar applies to other MUSTs citing informative references.
```suggestion
using the `verificationMethod` specified. The contents of the value SHOULD be a
```

Reply:
We do specify a subset of the multibase values in https://w3c.github.io/vc-data-integrity/#resource-integrity (but might have to tweak the language to refer to that section now). I'm also fine w downgrading to a SHOULD, but am concerned that if we do that, we open the door up to non-multibase values in a field whose datatype is multibase.

Reply:
If we are saying that `proofValue` is multibase... thats a normative dependency.

I don't see a way to remove the normative dependency, without opening the door to `proofValue`s that are not multibase... I also don't think thats a bad thing... its the crypto suite that should define the proof encoding, not the data integrity spec.

Reply:
Indeed, it should be perfectly legal for a cryptosuite to define its `proofValue` output as being pure base64url with no prefix.

Reply:
Use multibase enables compression when using other tools like CBOR-LD. There should always be a header and a single literal datatype (such that more than one context is not needed with potentially different term mapping definitions) to encapsulate the data. This enables flexibility in the face of future evolution whilst reducing overhead and preserving features in other layers.

So with a `DataIntegrityProof` in this version of the spec, the `proofValue` should always be encoded using multibase, i.e., there should be a header and a specific base to be used, as defined by the cryptosuite.

Reply:
@TallTed wrote:
> unless we replicate sufficient portion of the [[?MULTIBASE]] spec that this replication may be relied on for the MUST.

This is what the newest changes to this PR does, based on the request from @selfissued.

Reply:
It sounds like @dlongley is saying that there needs to be a normative dependency on multibase here:
> So with a DataIntegrityProof in this version of the spec, the proofValue should always be encoded using multibase, i.e., there should be a header and a specific base to be used, as defined by the cryptosuite.

Removing that dependency is exactly the problem that this PR needs to be addressing, which it hasn't yet done.

Reply:
> It sounds like @dlongley is saying that there needs to be a normative dependency on multibase here

No, I don't believe that is what he is saying.

You said this before:
> And that it would be replaced by in-spec statements that these values are strings beginning with either "u" or "z" and followed respectively by either a base64url-encoded value or a base58btc-encoded value.... Note that there's already text in the spec at https://w3c.github.io/vc-data-integrity/#resource-integrity defining these uses of the "u" and "z" prefixes. Promote these to be normative and ditch the reference to the individual Internet Draft.

Those references are now normative (which is what you asked for), and the references to the Multibase/Multihash I-Ds at IETF are now non-normative (which is what the group agreed to at TPAC).

> Removing that dependency is exactly the problem that this PR needs to be addressing, which it hasn't yet done.

Where is the normative dependency that you are alluding to?